Readme.md

```diff
@@ -0,0 +1,119 @@
+## Product overview 
+ The MRM-api is the backbone of a tool to facilitate room management. It enables  capturing of feedback based on room usage and analyse usage statistics.The MRM-api provides capability to register rooms and give feedback.
+
+## Development set up
+- Install python 3
+    - ##### Mac
+        -  Install Brew in Terminal
+        ```
+      /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        ```
+        -   Then Install Python 3
+        ```
+        brew install python3
+        ```
+    - ##### Windows
+        - Download installation setup
+        ```
+        https://www.python.org/downloads/
+        ```
+        - Click the icon labeling the  Python file  and follow the prompts
+        ```
+         for example  python-3.6.2.exe.
+        ```
+    - ##### Linux
+        - Run the following command in the terminal
+        ```
+        sudo apt-get update
+       ```
+        ```
+        sudo apt-get install python3.6
+        ```
+        
+- Install postgres
+    - ##### Mac
+    
+        ```
+        Follow guidelines  at this link (https://gist.github.com/sgnl/609557ebacd3378f3b72).
+        ```
+
+    - ##### Windows
+        ```
+        Follow guidelines  at this link (http://www.postgresqltutorial.com/install-postgresql/).
+        ```
+ 
+    - ##### Linux
+       ```
+       Follow guidelines  at this link (http://postgresguide.com/setup/install.html).
+       ```
+
+        
+- Install python install package (pip) manager 
+    ```
+        For guidelines to install pip for your Operating system follow this link (https://pip.pypa.io/en/stable/installing/)
+    ```
+- Install virtualenv
+    ```
+    pip install virtualenv
+    ```
+- Clone the mrm-api repo and cd into it
+    ```
+    git clone https://github.com/andela/mrm_api.git
+    ```
+- Create virtual env
+    ```
+    virtualenv --python=python3 venv
+    ```
+- Activate virtual env
+    ```
+    source venv/bin/activate
+    ```
+- Install dependencies
+    ```
+    pip install -r requirements.txt
+    ```
+- Create Application environment variables and save them in .env file
+    ```
+    export APP_SETTINGS="Development" # set app Enviroment.
+    export SECRET_KEY="some-very-long-string-of-random-characters"
+    export DEV_DATABASE_URL="" # Db for Development.
+    export TEST_DATABASE_URL="" # Db for Testing
+    export DATABASE_URL="" # Db for Production
+    ```
+- Running migrations
+    - Initialize  Database Structure. Run the command below :
```

Comment:
I am assuming while a user cloned the app, should he init the db, I believe the migrations folder will exist and will be part of the source code, the only requirement then would be to run migrations or make migrations, not to initialize